stuff/pipelines/producer_coder_of_month.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+'''Send message to coder_month_client'''
+
+
+import argparse
+import logging
+import os
+import sys
+import json
+from typing import Optional
+from rabbitmq_database import get_coder_of_the_month
```

Comment:
```suggestion

from rabbitmq_database import get_coder_of_the_month
```

idealmente hay que poner una línea separando las librerías incluidas y las de pypi.